examples/slash_commands/main.go

```diff
@@ -402,22 +416,24 @@ func main() {
 
 	if *RemoveCommands {
 		log.Println("Removing commands...")
-		// // We need to fetch the commands, since deleting requires the command ID.
-		// // We are doing this from the returned commands on line 375, because using
-		// // this will delete all the commands, which might not be desirable, so we
-		// // are deleting only the commands that we added.
+		// Uncomment the following code to delete every command in the application.
+		// You must also change newCommands (line 430) to registeredCommands.
+		//
+		// Fetch the created command IDs from the Discord API.
 		// registeredCommands, err := s.ApplicationCommands(s.State.User.ID, *GuildID)
```

Comment:
In addition to my comment on `registered`, I think it would be also fair to replace `:=` with `=`, due to the fact that when this line will be uncommented both variables will be shadowed.

It's not a critical flaw, but since this example is copied a lot, it would be good to fix it.